src/main/java/net/minecraftforge/client/event/FOVModifierEvent.java

```diff
@@ -24,35 +24,35 @@
 import net.minecraft.util.math.MathHelper;
 import net.minecraftforge.eventbus.api.Event;
 
-public class FOVUpdateEvent extends Event
+public class FOVModifierEvent extends Event
 {
     private final PlayerEntity entity;
     private final float fov;
     private float newfov;
```

Comment:
Might want to rename this field, too.

Reply:
See my previous comment on field/parameter renaming: https://github.com/MinecraftForge/MinecraftForge/pull/7255#discussion_r471101124.